packages/design-picker/src/available-designs-config.json

```diff
@@ -433,7 +433,7 @@
 			"categories": [ "featured" ],
 			"is_premium": false,
 			"is_fse": false,
-			"is_alpha": true,
+			"is_alpha": false,
```

Comment:
You can remove it entirely.

Reply:
Thanks so much @alshakero! I removed it :)